recipes/ckan-deploy.rb

```diff
@@ -70,6 +70,10 @@
 # Install selected revision of CKAN core
 #
 
+execute "Pin setuptools version" do
+	command "#{virtualenv_dir}/bin/pip install setuptools==44.1.0"
```

Comment:
seems like this should be a variable in our opsworks cfn vars.

Reply:
Eh, this version is the sweet spot that works on both Python 2 and 3. It can't be updated any further without ditching Python 2 entirely - at which point we can probably drop the pin and just use default behaviour.

Reply:
hmm, should it be a boolean flag then for 'pinning'?